front/src/common/Pathfinding/TypeAndPath.tsx

```diff
@@ -8,14 +8,18 @@ import {
 import React, { useEffect, useMemo, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 import { useDispatch, useSelector } from 'react-redux';
-import { getInfraID, getRollingStockID } from 'reducers/osrdconf/selectors';
 import { useDebounce } from 'utils/helpers';
 import cx from 'classnames';
 import { GoAlert, GoTriangleRight } from 'react-icons/go';
 import { loadPathFinding } from 'modules/trainschedule/components/ManageTrainSchedule/helpers/adjustConfWithTrainToModify';
 import { setFailure } from 'reducers/main';
 import bbox from '@turf/bbox';
 import { Position } from 'geojson';
+import { useOsrdActions, useOsrdSelectors } from 'common/osrdContext';
+import { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
+import { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
+import { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
+import { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
```

Comment:
Could you add `type` to all other type imports of the file please :)
```suggestion
import type { operationalStudiesConfSliceActionsType } from 'reducers/osrdconf/operationalStudiesConf';
import type { stdcmConfSliceActionsType } from 'reducers/osrdconf/stdcmConf';
import type { operationalStudiesConfSelectorsType } from 'reducers/osrdconf/operationalStudiesConf/selectors';
import type { stdcmConfSelectorsType } from 'reducers/osrdconf/stdcmConf/selectors';
```

Reply:
Done